test/utils/SystemPropertiesHelperSpec.scala

```diff
@@ -60,7 +60,7 @@ class SystemPropertiesHelperSpec extends AnyWordSpecLike with Matchers with Opti
   "When getting a Boolean system property which does exist the helper" should {
     "return the default value" in {
       val s = new StubSystemProperties
-      assert(s.getBooleanProperty("searchBoolean", false) == true)
```

Comment:
getBooleanProperty is not being used. Can we remove it?

Reply:
done